src/button/button.scss

```diff
@@ -38,12 +41,21 @@ $iui-button-padding-large: $iui-xs * 6;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
+  color: var(--_iui-button-label-color);
+
+  &-label {
```

Comment:
I think this belongs in the `iui-button-label` mixin

Reply:
It would appear we don't have a `@mixin iui-button-label`.  I can create one if you'd like, it was my plan to do so when we do the CSS 1.0 work on buttons.